.github/workflows/ci.yml

```diff
@@ -51,13 +47,23 @@ jobs:
           git clone https://github.com/avantifellows/plio-frontend/
           cd plio-frontend/
 
+          # check branch and switch to branch if exists
+          if [ `git branch --list --remote origin/${{ github.head_ref }}` ]
```

Comment:
<img width="741" alt="Screenshot 2021-12-07 at 12 54 03 PM" src="https://user-images.githubusercontent.com/20169829/144984679-86fb6b04-5d34-4b21-b409-a59dfa0cbb41.png">

Here, the `head_ref` will only be available if this workflow is being run on a pull request. What happens when we merge to prod, will the cypress CI runs won't happen?

It's another story if we don't want them to run when merging branches.

Reply:
I'm guessing during pushing to main/prod, as the `head_ref` will not be available, the code will just pull the default branch, right?

Reply:
That's a good point. I just checked and it seems to be returning empty value in case of direct push to the branch (without a pull request).

Since because of the empty value, the if condition will be false as it won't find a valid remote branch, hence it will go with default, which in our case is master. So I believe you're right that it will still run successfully.